order/buyer/receive-goods.md

```diff
@@ -0,0 +1,5 @@
+---
+description: How to annouce the buyer has received goods
+---
+
+# Receive goods
```

Comment:
This is an empty page?

Reply:
To be done under TC-5385 Create API v2 manual for buyer order process (part 2)